java-frontend/src/test/java/org/sonar/java/resolve/SymbolTableTest.java

```diff
@@ -1222,8 +1223,7 @@ public void infer_stream_types_on_chained_map() {
 
     IdentifierTree map = result.referenceTree(63, 8);
     MethodJavaType methodJavaType = (MethodJavaType) map.symbolType();
-    // expression type is correctly infered but method type is not recomputed and thus is still defered
-    assertThat(methodJavaType.resultType.isTagged(JavaType.DEFERRED)).isTrue();
+    assertThat(methodJavaType.resultType.isTagged(JavaType.DEFERRED)).isFalse();
```

Comment:
Maybe we can also have a positive assertion ? testing what is expected not only what is not expected ?